src/solc_0.8/raffle/GenericRaffle.sol

```diff
@@ -112,6 +131,8 @@ contract GenericRaffle is ERC721EnumerableUpgradeable, OwnableUpgradeable, Reent
         require(indexWave > 0, "Contract is not configured");
         require(_msgSender() == allowedToExecuteMint, "Not allowed");
         require(paused == 0, "Contract is paused");
+        require(_wallet != address(0x0), "Wallet is zero address");
```

Comment:
you can just use `address(0)` (let's be homogeneous)

Reply:
Done.